person.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# class for library
+class Person
+  # constructor method
+  def initialize(id, name = 'unknown', parent_permission: true)
```

Comment:
- [ ] you missed providing the `name` parameter for the constructor function please kindly add this :+1: 
- [ ] Also i see that you are passing in the `id` as a parameter, this should be auto generated. see the code snippet below as a guide
- code snippet
```ruby
  def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
```

Reply:
@Efiamotu-1, thank you for the review. However, I do not understand this comment " You missed providing the name parameter for the constructor function please kindly add this 👍". Could you clarify the issue? As you can see from the screenshot you sent, the name parameter is already added in the constructor. `def initialize(id, name = 'unknown', parent_permission: true)`

Reply:
sorry @Kanyorok i meant to type `age`

Reply:
Ooh, thank you @Efiamotu-1. I am fixing the issue.